tests/rule_based_profiler/test_profiler_user_workflows.py

```diff
@@ -250,12 +250,10 @@ def test_alice_profiler_user_workflow_single_batch(
     # Load data context
     data_context: DataContext = alice_columnar_table_single_batch_context
     # Load profiler configs & loop (run tests for each one)
-    profiler_config: str = alice_columnar_table_single_batch["profiler_config"]
+    yaml_config: str = alice_columnar_table_single_batch["profiler_config"]
```

Comment:
This was originally named `profiler_config`, which conflicts with the new constructor arg for `Profiler`. Let me know if we can be more descriptive here or if this suffices.

Reply:
Could you explain this a bit more? It looks like we're able to use the `profiler_config` name with no problem e.g. on L273 and L385. Why do we need to rename this?

Reply:
Per the changes made in this PR, the new arg in the `Profiler` constructor is called `profiler_config` (which is of type `dict`). Prior to the changes made, `profiler_config` on L253 was a `str` that was ultimately used to generate the appropriate `dict` passed into the constructor.

I've renamed `full_profiler_config_dict` to just `profiler_config` on L256 so that the parameter and argument names match (i.e. `validator=validator`). Let me know if that clarifies things

Reply:
Ah I see - makes sense!